src/features/game/expansion/components/IslandList.tsx

```diff
@@ -0,0 +1,98 @@
+import React from "react";
+
+import lock from "assets/skills/lock.png";
+import { useLocation, useNavigate, useParams } from "react-router-dom";
+import { OuterPanel } from "components/ui/Panel";
+import { getBumpkinLevel } from "features/game/lib/level";
+import { Bumpkin } from "features/game/types/game";
+
+const CONTENT_HEIGHT = 380;
+
+interface Island {
+  name: string;
+  levelRequired: number;
+  path: string;
+}
+
+interface Props extends Island {
+  bumpkin: Bumpkin | undefined;
+}
+
+const Island = ({ name, levelRequired, path, bumpkin }: Props) => {
+  const navigate = useNavigate();
+  const cannotNavigate =
+    !bumpkin || getBumpkinLevel(bumpkin.experience) < levelRequired;
+  return (
+    <>
+      {cannotNavigate ? (
+        <div>
+          <OuterPanel className="flex relative items-center py-2 mb-1 opacity-50">
+            <div className="flex-1 flex flex-col justify-center">
+              <span className="text-sm">{name}</span>
+              <div className="flex items-center">
+                <span
+                  className="bg-error border text-xxs p-1 rounded-md"
+                  style={{ lineHeight: "10px" }}
+                >
+                  Lvl {levelRequired}
+                </span>
+
+                <img src={lock} className="h-4 ml-1" />
+              </div>
+            </div>
+          </OuterPanel>
+        </div>
+      ) : (
+        <div onClick={() => navigate(path)}>
+          <OuterPanel className="flex relative items-center py-2 mb-1 cursor-pointer hover:bg-brown-200">
+            <div className="flex-1 flex flex-col justify-center">
+              <span className="text-sm">{name}</span>
+            </div>
+          </OuterPanel>
+        </div>
+      )}
+    </>
+  );
+};
+
+export const IslandList = ({ bumpkin }: { bumpkin: Bumpkin | undefined }) => {
+  const { id } = useParams();
+  const location = useLocation();
+  const islands: Island[] = [
+    {
```

Comment:
<img width="520" alt="Screen Shot 2022-10-12 at 1 04 20 pm" src="https://user-images.githubusercontent.com/11745561/195232019-db45a45d-233b-4474-aa74-2996a41db4df.png">

It would be amazing to see some icons next to these items so it is consistent with the Building List. It should add a bit of vibrance to this list move.

Goblin Retreat - Goblin Icon
Helios - Human Icon
Snow Kingdom - ???

Reply:
Snow Kingdom - 2 Snowball ⚪ (Big and smaller) or Snow Crystal ❄️ or Snow Monster ☃️ Icon?

Reply:
![snow_icons](https://user-images.githubusercontent.com/11745561/195488122-e1ccfbaf-9b8b-4905-8d4c-7700c1096dda.png)

Good idea @hujikol , we actually have these unused snow assets from earlier in the project. They could be useful @Rofrtd @HBandesh

Reply:
@adamhannigan , we have one more option in it - Farm. I was thinking for helios we can use merchant image (as a player would be there to trade - weather buying seeds/selling cakes) and for Farm we can use human. Thoughts?

Reply:
@HBandesh That sounds good to me 👍